### ISSUE TITLE
Can you add for AkelPad?

### ISSUE THREAD

**Issue by azjio** [Predelnik/DSpellCheck#200]
Do I use AkelPad? It uses aspell.exe with a 57Mb dictionary. It works slowly. It is impossible to use 2 dictionaries at the same time.
Can you make a plugin for AkelPad?

**Comment by Predelnik** [comment]
Since I abstracted away at least some interaction with Notepad++ for testing purposes, it is not absolutely impossible but still would not be easy. I need to look into it in detail a bit later to say if there's even a chance.

**Comment by azjio** [comment]
[SpellCheck.zip](http://akelpad.sourceforge.net/files/plugs/SpellCheck.zip) - in this archive AkelPad plugin source. Maybe it somehow helps. This archive is on the AkelPad [plugins home page](http://akelpad.sourceforge.net/ru/plugins.php).

**Comment by azjio** [comment]
This function will highlight words (see source QSearchDlg.c)
`static void CallPluginFuncW(const wchar_t* cszFuncW, void* pParams)`
This is a call to an external "Coder plugin". 
See file "AkelPad\AkelFiles\Docs\Coder-Eng.txt", line 55
`Call("Coder::HighLight", 2, "#RRGGBB", "#RRGGBB", FLAGS, FONTSTYLE, ID, "TEXT", TEXTLENGTH)`
These parameters are passed in the structure.
That is, instead of drawing wavy lines, you can enable the underline style and make the text red.

It would be useful to display the words with errors in the console, and then clicking on the word makes a jump to the line with this word.
```
[Line: 9, pos: 59] comsole (hint: console)
[Line: 10, pos: 3] hihlight (hint: highlight)
```

**Comment by anonimuos** [comment]
Yes please it would be so nice to have 64-bit SpellCheck.dll or 64-bit DSpellCheck for 64-bit AkelPad
